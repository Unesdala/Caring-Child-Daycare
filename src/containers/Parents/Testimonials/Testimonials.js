import React, { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultTestimonialsContent from './TestimonialsContent';

export class Testimonials extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Parent Testimonials'); }

  render() {
    return (
      <div className="page-content">
        <DefaultTestimonialsContent />
      </div>
    );
  }
}

export default Testimonials;
