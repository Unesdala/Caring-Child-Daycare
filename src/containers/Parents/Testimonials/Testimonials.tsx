import { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultTestimonialsContent from './TestimonialsContent';

export class Testimonials extends Component {
  commonUtils: { setTitleAndScroll: (pageTitle: string, width: number) => void };

  constructor(props: Record<string, unknown>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Parent Testimonials', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultTestimonialsContent />
        </div>
      </>
    );
  }
}

export default Testimonials;
