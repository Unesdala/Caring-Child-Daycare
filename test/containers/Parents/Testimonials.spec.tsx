/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Testimonials } from 'src/containers/Parents/Testimonials/Testimonials';
import TestimonialsContent from 'src/containers/Parents/Testimonials/TestimonialsContent';

describe('Testimonials', () => {
  const wrapper = shallow(<Testimonials />);
  it('renders', () => {
    expect(wrapper.find(TestimonialsContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(TestimonialsContent).dive().find('main.testimonials-main--content').exists()).toBe(true);
  });
});