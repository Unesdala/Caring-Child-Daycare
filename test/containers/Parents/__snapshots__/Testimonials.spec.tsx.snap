// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials renders 1`] = `ShallowWrapper {}`;
